queries.sql

```diff
@@ -54,4 +48,40 @@ GROUP BY species;
 SELECT species, AVG(escape_attempts)
 FROM animals
 WHERE date_of_birth >= '1990-01-01' AND date_of_birth <= '2000-12-31'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+
+SELECT name FROM animals
+JOIN owners ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Melody Pond';
+
+SELECT animals.name FROM animals
+JOIN species ON animals.species_id = species.id
+WHERE species.name = 'Pokemon';
+
+
+SELECT owners.full_name, animals.name
+FROM owners
+LEFT JOIN animals ON owners.id = animals.owner_id;
+
+SELECT species.name, COUNT(animals.id) as count
+FROM species
+LEFT JOIN animals ON species.id = animals.species_id
+GROUP BY species.id;
+
+SELECT animals.name
+FROM animals
+JOIN species ON animals.species_id = species.id
+JOIN owners ON animals.owner_id = owners.id
+WHERE species.name = 'Digimon' AND owners.full_name = 'Jennifer Orwell';
```

Comment:
The result of the corresponding screenshot to this query in your PR is not correct. **Jennifer Orwell** has one **Digimon**, *Gabumon*  and your results show none. Kindy fix this error.